## Betacoronavirus


Betacoronaviruses (β-CoVs or Beta-CoVs) are one of four genera (Alpha-, Beta-, Gamma-, and Delta-) of coronaviruses. It is in the subfamily Orthocoronavirinae in the family Coronaviridae, of the order Nidovirales. They are enveloped, positive-sense, single-stranded RNA viruses of zoonotic origin. The coronavirus genera are each composed of varying viral lineages with the betacoronavirus genus containing four such lineages: A, B, C, D. In older literature, this genus is also known as group 2 coronaviruses

The Beta-CoVs of the greatest clinical importance concerning humans are OC43 (which can cause the common cold) and HKU1 of the A lineage, SARS-CoV and SARS-CoV-2 (which causes the disease COVID-19) of the B lineage,[3] and MERS-CoV of the C lineage. MERS-CoV is the first betacoronavirus belonging to lineage C that is known to infect humans.[4][5]

The Alphacoronavirus and Betacoronavirus genera descend from the bat gene pool.[6][7][8]

## Virology


Alpha- and Betacoronaviruses mainly infect bats, but they also infect other species like humans, camels, and rabbits.[6][7][8][9] Beta-CoVs that have caused epidemics in humans generally induce fever and respiratory symptoms. They include:

+ SARS-CoV, SARS.
+ MERS-CoV, MERS.
+ SARS-CoV-2, COVID-19.

## Sequence


Coronaviruses have a large genome size that ranges from 26 to 32 kilobases. Within the genus Betacoronavirus (Group 2 CoV), four lineages (A, B, C, and D) are commonly recognized.[11]

+ Lineage A (subgenus Embecovirus) includes HCoV-OC43 and HCoV-HKU1 (various species)
+ Lineage B (subgenus Sarbecovirus) includes SARSr-CoV (which includes all its strains such as SARS-CoV, SARS-CoV-2, and Bat SL-CoV-WIV1)
+ Lineage C (subgenus Merbecovirus) includes Tylonycteris bat coronavirus HKU4 (BtCoV-HKU4), Pipistrellus bat coronavirus HKU5 (BtCoV-HKU5), and MERS-CoV (various species)
+ Lineage D (subgenus Nobecovirus) includes Rousettus bat coronavirus HKU9 (BtCoV-HKU9)[12]

Genetic recombination can occur when two or more viral genomes are present in the same host cell. The dromedary camel Beta-CoV HKU23 exhibits genetic diversity in the African camel population.[16] Contributing to this diversity are several recombination events that had taken place in the past between closely related Beta-CoVs of the subgenus Embecovirus.[16] Also the Beta-CoV, Human SARS-CoV, appears to have had a complex history of recombination between ancestral coronaviruses that were hosted in several different animal groups.[17][18]


## References


[3] "Phylogeny of SARS-like betacoronaviruses". nextstrain. Retrieved 18 January 2020.

[4] ProMED. MERS-CoV–Eastern Mediterranean (06) (http://www.promedmail.org/)

[5] Memish, Z. A.; Zumla, A. I.; Al-Hakeem, R. F.; Al-Rabeeah, A. A.; Stephens, G. M. (2013). "Family Cluster of Middle East Respiratory Syndrome Coronavirus Infections". New England Journal of Medicine. 368 (26): 2487–94. doi:10.1056/NEJMoa1303729. PMID 23718156.
 
[6] Woo, P. C.; Wang, M.; Lau, S. K.; Xu, H.; Poon, R. W.; Guo, R.; Wong, B. H.; Gao, K.; Tsoi, H. W.; Huang, Y.; Li, K. S.; Lam, C. S.; Chan, K. H.; Zheng, B. J.; Yuen, K. Y. (2007). "Comparative analysis of twelve genomes of three novel group 2c and group 2d coronaviruses reveals unique group and subgroup features". Journal of Virology. 81 (4): 1574–85. doi:10.1128/JVI.02182-06. PMC 1797546. PMID 17121802.

[7] Lau, S. K.; Woo, P. C.; Yip, C. C.; Fan, R. Y.; Huang, Y.; Wang, M.; Guo, R.; Lam, C. S.; Tsang, A. K.; Lai, K. K.; Chan, K. H.; Che, X. Y.; Zheng, B. J.; Yuen, K. Y. (2012). "Isolation and characterization of a novel Betacoronavirus subgroup A coronavirus, rabbit coronavirus HKU14, from domestic rabbits". Journal of Virology. 86 (10): 5481–96. doi:10.1128/JVI.06927-11. PMC 3347282. PMID 22398294.

[8] Lau, S. K.; Poon, R. W.; Wong, B. H.; Wang, M.; Huang, Y.; Xu, H.; Guo, R.; Li, K. S.; Gao, K.; Chan, K. H.; Zheng, B. J.; Woo, P. C.; Yuen, K. Y. (2010). "Coexistence of different genotypes in the same bat and serological characterization of Rousettus bat coronavirus HKU9 belonging to a novel Betacoronavirus subgroup". Journal of Virology. 84 (21): 11385–94. doi:10.1128/JVI.01121-10. PMC 2953156. PMID 20702646.

[9] Zhang, Wei; Zheng, Xiao-Shuang; Agwanda, Bernard; Ommeh, Sheila; Zhao, Kai; Lichoti, Jacqueline; Wang, Ning; Chen, Jing; Li, Bei; Yang, Xing-Lou; Mani, Shailendra; Ngeiywa, Kisa-Juma; Zhu, Yan; Hu, Ben; Onyuok, Samson Omondi; Yan, Bing; Anderson, Danielle E.; Wang, Lin-Fa; Zhou, Peng; Shi, Zheng-Li (24 October 2019). "Serological evidence of MERS-CoV and HKU8-related CoV co-infection in Kenyan camels". Emerging Microbes & Infections. 8 (1): 1528–1534. doi:10.1080/22221751.2019.1679610. PMC 6818114. PMID 31645223.


[11]:"Woo, Huang & Lau 2010". In all members of Betacoronavirus subgroup A, a haemagglutinin esterase (HE) gene, which encodes a glycoprotein with neuraminate O-acetyl-esterase activity and the active site FGDS, is present downstream to ORF1ab and upstream to S gene (Figure 1).

[16] Diversity of Dromedary Camel Coronavirus HKU23 in African Camels Revealed Multiple Recombination Events among Closely Related Betacoronaviruses of the Subgenus Embecovirus. So RTY, et al. J Virol. 2019. PMID: 31534035

[17] Stanhope MJ, Brown JR, Amrine-Madsen H. Evidence from the evolutionary analysis of nucleotide sequences for a recombinant history of SARS-CoV. Infect Genet Evol. 2004 Mar;4(1):15-9. PMID: 15019585

[18] Zhang XW, Yap YL, Danchin A. Testing the hypothesis of a recombinant origin of the SARS-associated coronavirus. Arch Virol. 2005 Jan;150(1):1-20. Epub 2004 Oct 11. PMID: 15480857


